charts/pokt-portal-v2/templates/deployment.yaml

```diff
@@ -108,13 +113,7 @@ spec:
       {{- end }}
       volumes:
         {{- if .Values.middleware.enabled }}
-        {{- if .Values.externalSecrets.enabled }}
-        - name: external-secrets
-          secret:
```

Comment:
How are we thinking about ingesting the secrets ? Not sure If I understand the proposition behind removing this volume mount

Reply:
The way this secret was set up it assumed loading via env vars. We have discussed moving to a templated config.json. Saving this enhancement for a another PR: https://pokt.height.app/portal-team/T-19189?task=T-19198